portal-web/test/functional/com/liferay/portalweb/macros/CookiePreferenceHandlingAdmin.macro

```diff
@@ -1,5 +1,35 @@
 definition {
 
+	@description = "Disable Cookie Preference handling"
```

Comment:
Please expand the description indicating it disables the cookie settings at Instance level

Reply:
Done